graylog2-web-interface/src/contexts/CurrentUserPreferencesProvider.jsx

```diff
@@ -0,0 +1,27 @@
+// @flow strict
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import { get } from 'lodash';
+
+import { useStore } from 'stores/connect';
+import CombinedProvider from 'injection/CombinedProvider';
+import UserPreferencesContext from './UserPreferencesContext';
+
+const { CurrentUserStore } = CombinedProvider.get('CurrentUser');
```

Comment:
What is the reason to use `CombinedProvider.get(` here? Compared with `import CurrentUserStore from 'stores/users/CurrentUserStore';` the only difference is that using the `CombinedProvider`, `preferences` is not `undefined` initially.